website/addons/wiki/model.py

```diff
@@ -51,7 +55,58 @@ def subscribe_on_write_permissions_revoked(node):
         wiki_utils.migrate_uuid(node, wiki_name)
 
 
-def build_wiki_url(node, label, base, end):
+def wiki_updates(node, user, **context):
+    w_url = furl(node.absolute_url)
+    try:
+        w_url.path = context.pop('path')
+    except KeyError:
+        print "Please return path."
```

Comment:
Actually, if `path` is required, lets add it as a required keyword argument instead.